diffkemp/test_tools/module_tools.py

```diff
@@ -0,0 +1,29 @@
+"""Tools for working with modules used by both diffkemp and diffkabi tests."""
```

Comment:
I'd put this directly to the tests/regression directory.

Reply:
I don't exactly know how to make Python find the file there. (I could turn the whole tests directory into a module, but then it could cause a conflict when including the diffkemp repository into another project.)

Reply:
I think it could be fine, because if someone includes the repo into another project, he would just use the 'diffkemp' package.